samples/subsys/mgmt/updatehub/README.rst

```diff
@@ -252,10 +435,12 @@ following command:
 
     updatehub run
 
-And then wait. The board will ping the server, check if there are any new
+And then wait.  The board will probe the server, check if there are any new
 updates, and then download the update package you've just created. If
 everything goes fine the message ``Image flashed successfully, you can reboot
-now`` will be printed on the terminal.
+now`` will be printed on the terminal.  If you are using the ``Pooling`` mode
```

Comment:
same here, Pooling -> Polling, can you search and replace all of these (just in case there are more)

Reply:
Fixed!